game.js

```diff
@@ -14,7 +14,7 @@ function start() {
     return 0.42;
   }
   setTimeout(() => {
-      
```

Comment:
lol
somehow it got in there!

![](https://media.giphy.com/media/26FPF032XnQCUsw4E/giphy.gif)